src/pages/MenuPage.js

```diff
@@ -38,96 +95,66 @@ export class MenuPage extends React.Component {
     database.removeListener(this.databaseListenerId);
   }
 
+  /**
+   * Method will try to match stringified customData object for current store
+   * against keys in MODULES, will return values of keys in MODULES
+   * that match: customData[keyInModules]data === 'true'
+   * @param {string}  customDataAsString  Store custom data as string
+   * @return {array}  result              The maxOrdersPerPeriod of the orderType
+   */
+  getUsedModules = customDataAsString => {
```

Comment:
Do you think this would ever be used elsewhere? If so maybe a utility method? (would require some reshuffling. Maybe having the method return a string of modules, iterate through afterward adding the button from `MODULES` to `modules`.

Reply:
i'll try to deal with this also in that issue, will create it later today

Reply:
https://github.com/openmsupply/mobile/issues/880